specs/SIGNATURE_SPEC.md

```diff
@@ -129,6 +129,11 @@ Gyp4apdU7AXEwysEQIb034aPrTlpmxh90SnTZFs2DHOvCjCPPAmoWfuQUwPhSPRb
 
 For instructions on using the `bundle` for verification, see [USAGE.md](../USAGE.md#verify-a-signature-was-added-to-the-transparency-log).
 
+* `rfc3161timestamp` string
+
+  This OPTIONAL property contains a JSON formatted `RFC3161Timestamp` bundle containing the timestamp response from a
```

Comment:
I suggest to avoid using the term bundle here, as you mentioned it can be confusing.

Reply:
+1 here it's not a bundle anymore of cert/sig as well, so it's just the RFC 3161 response, I think?

Reply:
Correct, removed.